server/src/main/java/io/whitefox/api/deltasharing/server/DeltaSharesApiImpl.java

```diff
@@ -49,7 +65,20 @@ public Response getTableChanges(
   @Override
   public Response getTableMetadata(
       String share, String schema, String table, String startingTimestamp) {
-    return Response.ok().build();
+    // TODO set header(deltaSharingCapabilities, responseFormat)
```

Comment:
I think this TODO is outdated

Reply:
Nope, it's the last thing that I have to do